games/devtest/mods/lighting/set_lightIntensity.lua

```diff
@@ -0,0 +1,141 @@
+
+local lightIntensity_sections = {
+  {n = "colorOffset", d = "Color offset",
+    entries = {
+      {n = "r", d = "Red color offset", min = -1, max = 2},
+      {n = "g", d = "Green color offset", min = -1, max = 2},
+      {n = "b", d = "Blue color offset", min = -1, max = 2},
+    }
+  },
+  {n = "colorRatioCoef", d = "Color day-night ratio coefficient",
+    entries = {
+      {n = "r", d = "Red color day-night ratio coefficient", min = -1e-3, max = 2e-3},
+      {n = "g", d = "Green color day-night ratio coefficient", min = -1e-3, max = 2e-3},
+      {n = "b", d = "Blue color day-night ratio coefficient", min = -1e-3, max = 2e-3},
+    }
+  }
+}
+
+local function dump_lightIntensity(lightIntensity)
```

Comment:
Why does this need a special dump? Is there a problem with using the normal `dump` or `dump2`?

Reply:
I have no idea. I have copied and reused the command `set_lighting` (https://github.com/minetest/minetest/blob/180ec92ef982d9fb5c6bdc789f381335f77823c1/games/devtest/mods/lighting/init.lua) with being happy, that I do not have to develop form spec myself.